sites/website/docusaurus.config.js

```diff
@@ -13,12 +13,19 @@ module.exports = {
             async: true,
         },
     ],
-    themes: ["@docusaurus/theme-live-codeblock"],
+    themes: [require.resolve("@docusaurus/theme-live-codeblock")],
     themeConfig: {
         navbar: {
             logo: {
                 alt: "FAST-DNA",
                 src: "img/logo.svg",
```

Comment:
This is a good candidate for the CDN `https://static.fast.design/assets/brand/fast-logo.svg` as an example and is achievable by placing in `/sites/site-utilities/statics/assets/brand/fast-logo.svg` upon merge to master ( know this is a feature branch ) it would deploy to CDN.